2664470#: debug1: sshd version OpenSSH_5.3p1
2664470#: debug1: read PEM private key done: type RSA
2664470#: debug1: private host key: #0 type 1 RSA
2664470#: Disabling protocol version 1. Could not load host key
2664470#: debug1: setgroups() failed: Operation not permitted
2664470#: debug1: rexec_argv[0]='/usr/sbin/sshd'
2664470#: debug1: rexec_argv[1]='-o'
2664470#: debug1: rexec_argv[2]='AuthorizedKeysFile=/var/lib/condor/execute/dir_2663996/tmpl2smld'
2664470#: debug1: rexec_argv[3]='-o'
2664470#: debug1: rexec_argv[4]='StrictModes=no'
2664470#: debug1: rexec_argv[5]='-o'
2664470#: debug1: rexec_argv[6]='UsePam=no'
2664470#: debug1: rexec_argv[7]='-o'
2664470#: debug1: rexec_argv[8]='PermitRootLogin=no'
2664470#: debug1: rexec_argv[9]='-o'
2664470#: debug1: rexec_argv[10]='PasswordAuthentication=no'
2664470#: debug1: rexec_argv[11]='-o'
2664470#: debug1: rexec_argv[12]='PidFile=/dev/null'
2664470#: debug1: rexec_argv[13]='-h'
2664470#: debug1: rexec_argv[14]='/var/lib/condor/execute/dir_2663996/tmpkDgwwO'
2664470#: debug1: rexec_argv[15]='-D'
2664470#: debug1: rexec_argv[16]='-e'
2664470#: debug1: rexec_argv[17]='-f'
2664470#: debug1: rexec_argv[18]='/dev/null'
2664470#: debug1: rexec_argv[19]='-d'
2664470#: debug1: rexec_argv[20]='-p'
2664470#: debug1: rexec_argv[21]='5001'
2664470#: Set /proc/self/oom_score_adj from 800 to -1000
2664470#: debug1: Bind to port 5001 on 0.0.0.0.
2664470#: Server listening on 0.0.0.0 port 5001.
2664470#: debug1: Bind to port 5001 on ::.
2664470#: Server listening on :: port 5001.
2664470#: debug1: Server will not fork when running in debugging mode.
2664470#: debug1: rexec start in 5 out 5 newsock 5 pipe -1 sock 8
2664470#: debug1: sshd version OpenSSH_5.3p1
2664470#: debug1: read PEM private key done: type RSA
2664470#: debug1: private host key: #0 type 1 RSA
2664470#: Disabling protocol version 1. Could not load host key
2664470#: debug1: setgroups() failed: Operation not permitted
2664470#: debug1: inetd sockets after dupping: 3, 3
2664470#: Connection from 2607:f720:1700:31d::61 port 32459
2664470#: debug1: Client protocol version 2.0; client software version OpenSSH_5.3
2664470#: debug1: match: OpenSSH_5.3 pat OpenSSH*
2664470#: debug1: Enabling compatibility mode for protocol 2.0
2664470#: debug1: Local version string SSH-2.0-OpenSSH_5.3
2664470#: debug1: list_hostkey_types: ssh-rsa
2664470#: debug1: SSH2_MSG_KEXINIT sent
2664470#: debug1: SSH2_MSG_KEXINIT received
2664470#: debug1: kex: client->server aes128-ctr hmac-md5 none
2664470#: debug1: kex: server->client aes128-ctr hmac-md5 none
2664470#: debug1: SSH2_MSG_KEX_DH_GEX_REQUEST received
2664470#: WARNING: /etc/ssh/moduli does not exist, using fixed modulus
2664470#: debug1: SSH2_MSG_KEX_DH_GEX_GROUP sent
2664470#: debug1: expecting SSH2_MSG_KEX_DH_GEX_INIT
2664470#: debug1: SSH2_MSG_KEX_DH_GEX_REPLY sent
2664470#: debug1: SSH2_MSG_NEWKEYS sent
2664470#: debug1: expecting SSH2_MSG_NEWKEYS
2664470#: debug1: SSH2_MSG_NEWKEYS received
2664470#: debug1: KEX done
2664470#: debug1: userauth-request for user slot4 service ssh-connection method none
2664470#: debug1: attempt 0 failures 0
2664470#: debug1: userauth-request for user slot4 service ssh-connection method publickey
2664470#: debug1: attempt 1 failures 0
2664470#: debug1: test whether pkalg/pkblob are acceptable
2664470#: debug1: temporarily_use_uid: 1004/1004 (e=1004/1004)
2664470#: debug1: trying public key file /var/lib/condor/execute/dir_2663996/tmpl2smld
2664470#: debug1: fd 4 clearing O_NONBLOCK
2664470#: debug1: matching key found: file /var/lib/condor/execute/dir_2663996/tmpl2smld, line 1
2664470#: Found matching RSA key: 35:6c:a6:7e:e7:a6:7c:65:ca:71:85:3e:93:06:03:59
2664470#: debug1: restore_uid: (unprivileged)
2664470#: Postponed publickey for slot4 from 2607:f720:1700:31d::61 port 32459 ssh2
2664470#: debug1: userauth-request for user slot4 service ssh-connection method publickey
2664470#: debug1: attempt 2 failures 0
2664470#: debug1: temporarily_use_uid: 1004/1004 (e=1004/1004)
2664470#: debug1: trying public key file /var/lib/condor/execute/dir_2663996/tmpl2smld
2664470#: debug1: fd 4 clearing O_NONBLOCK
2664470#: debug1: matching key found: file /var/lib/condor/execute/dir_2663996/tmpl2smld, line 1
2664470#: Found matching RSA key: 35:6c:a6:7e:e7:a6:7c:65:ca:71:85:3e:93:06:03:59
2664470#: debug1: restore_uid: (unprivileged)
2664470#: debug1: ssh_rsa_verify: signature correct
2664470#: Accepted publickey for slot4 from 2607:f720:1700:31d::61 port 32459 ssh2
2664470#: debug1: monitor_child_preauth: slot4 has been authenticated by privileged process
2664470#: debug1: SELinux support disabled
2664470#: User child is on pid 2664480
2664470#: debug1: Entering interactive session for SSH2.
2664470#: debug1: server_init_dispatch_20
2664470#: debug1: server_input_channel_open: ctype session rchan 0 win 2097152 max 32768
2664470#: debug1: input_session_request
2664470#: debug1: channel 0: new [server-session]
2664470#: debug1: session_new: session 0
2664470#: debug1: session_open: channel 0
2664470#: debug1: session_open: session 0: link with channel 0
2664470#: debug1: server_input_channel_open: confirm session
2664470#: debug1: server_input_global_request: rtype no-more-sessions@openssh.com want_reply 0
2664470#: debug1: server_input_channel_req: channel 0 request exec reply 1
2664470#: debug1: session_by_channel: session 0 channel 0
2664470#: debug1: session_input_channel_req: session 0 req exec
2664470#: debug1: session_new: session 0
2664470#: debug1: Received SIGCHLD.
2664470#: debug1: session_by_pid: pid 2664481
2664470#: debug1: session_exit_message: session 0 channel 0 pid 2664481
2664470#: debug1: session_exit_message: release channel 0
2664470#: Received disconnect from 2607:f720:1700:31d::61: 11: disconnected by user
2664470#: debug1: do_cleanup
2664470#: debug1: do_cleanup
Traceback (most recent call last):
  File "/var/lib/condor/execute/dir_2663996/condor_exec.exe", line 173, in <module>
    main(sys.argv[1:])
  File "/var/lib/condor/execute/dir_2663996/condor_exec.exe", line 169, in main
    performPlacement(inputfile,outputfile)
  File "/var/lib/condor/execute/dir_2663996/condor_exec.exe", line 132, in performPlacement
    pMover.server()
  File "/var/lib/condor/execute/dir_2663996/UDTMover.py", line 51, in server
    self.run()
  File "/var/lib/condor/execute/dir_2663996/DataMover.py", line 166, in run
    errhandler=self.stderrHandler)
  File "/var/lib/condor/execute/dir_2663996/TimedExec.py", line 100, in runTimedCmd
    stderr=errtype, stdout=outtype)
  File "/usr/lib64/python2.6/subprocess.py", line 642, in __init__
    errread, errwrite)
  File "/usr/lib64/python2.6/subprocess.py", line 1238, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
